plugins/remote-config/frontend/public/javascripts/countly.views.js

```diff
@@ -2,13 +2,332 @@
+        label: "#F5C900"
+    }
+
+    ];
+
+    var DATA_TYPES = {
+        d: CV.i18n("remote-config.type.d"),
+        n: CV.i18n("remote-config.type.n"),
+        bl: CV.i18n("remote-config.type.bl"),
+        s: CV.i18n("remote-config.type.s"),
+        l: CV.i18n("remote-config.type.l")
+    };
+
+    var ConditionStats = countlyVue.views.BaseView.extend({
+        template: '	<table class="cly-vue-remote-config-percentages-breakdown">\
+			<thead>\
+				<tr>\
+					<th class="cly-vue-remote-config-percentages-breakdown__sequence__heading bu-pl-2">\
+						#\
+					</th>\
+					<th class="cly-vue-remote-config-percentages-breakdown__heading">\
+						{{i18n("remote-config.condition")}}\
+					</th>\
+                    		<th class="cly-vue-remote-config-percentages-breakdown__heading">\
+						{{i18n("remote-config.percentage")}}\
+					</th>\
+				</tr>\
+			</thead>\
+			<tbody>\
+                <tr><td><div class="cly-vue-remote-config-percentages-breakdown__sequence bu-py-1 bu-px-2">1</div></td><td><div class="bu-is-align-items-center	bu-is-flex cly-vue-remote-config-percentages-breakdown__data bu-p-1 cly-vue-remote-config-percentages-breakdown__default-value"><span class="bu-ml-2 bu-mr-3">{{i18n("remote-config.default-value")}}</span><span class="cly-vue-remote-config-percentages-breakdown__default-value__value bu-py-1 bu-px-2">{{defaultValue.value}}</span></div></td><td>{{defaultValue.percentage}} {{i18n("remote-config.percent.of.total")}}</td></tr>\
+				<tr v-for="(condition, i) in conditions" :key="i">\
+                    <td><div class="cly-vue-remote-config-percentages-breakdown__sequence bu-py-1 bu-px-2">{{i+2}}</div>\
+                    </td>\
+					     <td><div class="bu-is-align-items-center	bu-is-flex cly-vue-remote-config-percentages-breakdown__data bu-p-1 cly-vue-remote-config-percentages-breakdown__condition" :style="{backgroundColor: condition.color}"><span><i class="ion ion-git" style="transform:rotate(90deg)"></i></span><span class="bu-ml-2 bu-mr-3">{{condition.name}}</span><span class="cly-vue-remote-config-percentages-breakdown__condition__value bu-py-1 bu-px-2">{{condition.value}}</span></div></td>\
+					<td>\
+                    {{condition.percentage}} {{i18n("remote-config.percent.of.total")}}\
+					</td>\
+				</tr>\
+			</tbody>\
+		</table>',
+        props: {
+            parameter: {
+                type: Object,
+                default: function() {
+                    return {};
+                }
+            }
+        },
+        computed: {
+            conditions: function() {
+                var conditions = [];
+                var conditionsMap = this.$store.getters["countlyRemoteConfig/parameters/conditionsMap"];
+                var self = this;
+                if (this.parameter.conditions.length > 0) {
+                    this.parameter.conditions.forEach(function(condition) {
+                        var conditionProperties = conditionsMap[condition.condition_id];
```

Comment:
ahhh so here you fetching the condition from the conditionsMap.
I think you can use .filter on the conditions array that already exists in vuex.
That should be fine i guess.